src/cultionet/models/unet_parts.py

```diff
@@ -480,7 +487,9 @@ def __init__(
             n_pools=1,
             n_stream_down=2,
             init_point_conv=init_point_conv,
-            double_dilation=double_dilation
+            dilations=[double_dilation],

```

Comment:
so dilations is a list of integer but you usually just have 1 element in the list 👍

Reply:
The default is a length-1 list when `res_block_type='res'` and `[1, 2]` when `res_block_type='resa'`.